test/pkg/firewall/firewall.go

```diff
@@ -0,0 +1,181 @@
+package firewall
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/openshift-kni/commatrix/pkg/consts"
+	"github.com/openshift-kni/commatrix/pkg/utils"
+)
+
+type Firewall struct {
+	ns           string
+	utilsHelpers utils.UtilsInterface
+}
+
+func New(ns string, utilsHelpers utils.UtilsInterface) *Firewall {
+	return &Firewall{
+		ns:           ns,
+		utilsHelpers: utilsHelpers,
+	}
+}
+
+func (firewall *Firewall) runBashCommandOnPod(debugPod *v1.Pod, command string, chrootDir string) ([]byte, error) {
```

Comment:
What is chrootDir?

Reply:
there is commands that we are ruuning on chroot /host

Reply:
Always to /host right? So maybe you can just have a bool saying if you need to change to root?